authors/urls.py

```diff
@@ -15,8 +15,24 @@
 """
 from django.urls import include, path
 from django.contrib import admin
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from rest_framework.permissions import AllowAny
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Authors Haven",
+      default_version='v1',
+      description="A social platform for the creative at heart.",
+      terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="contact@snippets.local"),
+      license=openapi.License(name="BSD License"),
+   ),
+   permission_classes=(AllowAny,),
+)
 
 urlpatterns = [
+    path('swagger/', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui'),
```

Comment:
@calvinpete I think you should change the path to `documentation/` instead of `swagger/`

Reply:
HE CAN AS WELL AS PUT IT AT '/' since there is nothing there @ja-odur

Reply:
Good point @MuhweziDeo so that when our application is loaded, the docs show up on the index route. I hope that makes sense.